website_multi_theme/models/ir_ui_view.py

```diff
@@ -45,3 +45,22 @@ def _customize_template_get_views(self, key, full=False, bundles=False):
             return views
         current_website = request.website
         return views.filtered(lambda v: v.website_id == current_website)
+
+    # Workaround for https://github.com/odoo/odoo/pull/24429
+    def search(self, domain, offset=0, limit=None, order=None, count=False):
+        if self.env.context.get('search_multi_website_snippet'):
+            website_id = self.env.context['search_multi_website_snippet']
+            domain += [
+                '|',
+                ('website_id', '=', website_id),
+                ('website_id', '=', False)
+            ]
+            _logger.debug('Updated domain: %s', domain)
+
+        res = super(IrUiView, self).search(
+            domain, offset=offset, limit=limit, order=order, count=count)
```

Comment:
I think you're forgetting to patch `order`

Reply:
I suggested [this](https://github.com/yelizariev/odoo/commit/52de9ae545506c8601e727c0cc80b52b4cce3707#r28729620) to him, and he put it [here](https://github.com/odoo/odoo/pull/24429/files#diff-0662b757930550c519324fd45a444369R54)

Reply:
Well, it's an enhancement there for giving more priority to specific website. OK then.

Reply:
Fixed and covered https://github.com/yelizariev/website/commit/4816f516a289a9ee57eb3b7186d065add1e29d8a